src/state/timer-state.js

```diff
@@ -1,5 +1,6 @@
 const Timer = require('./timer')
 const Mobbers = require('./mobbers')
+const ClearClipboard = require('../clear-clipboard')

```

Comment:
A minor thing: the usual convention is to only use UpperCamelCase naming for things that you should call `new` on. The timer and mobbers modules expose classes and you need to call `new` on them to use them properly. But this one doesn't need a `new` call.

Related to my other comment, this var could just be called `clipboard` since the act of clearing it is a function exposed by the module.